packages/terra-form/src/Field.scss

```diff
@@ -1,62 +1,80 @@
+@import '~terra-mixins';
+@import './cerner-mock-theme';
+
 :local {
-  .field,
-  .fieldset {
+  .field {
     border: 0;
-    margin-bottom: 1.428em;
+    margin-bottom: var(--terra-form-field-margin-bottom, 1.428em);
+    max-width: 1020px;
     padding: 0;
   }
 
+  // Remove this??
```

Comment:
Remove this (hah meta)

Reply:
Ah right. This was a reminded. With the newest uplift should we remove screen width specific stylings and leave responsiveness to a HOC using the Field?